frontend/src/static/js/v_summary.js

```diff
@@ -407,6 +408,11 @@ window.vSummary = {
         this.contributionBarColors = colors;
       });
     },
+    getFilteredAfterWatcherUpdated() {
+      if (this.isWatcherUpdated) {
+        this.getFiltered();
```

Comment:
instead of doing this, why not just return in `getFiltered` if `this.isWatcherUpdated` is false instead?